selfhost/parser.jakt

```diff
@@ -1502,6 +1517,11 @@ struct Parser {
                     parameter_complete = true
                 }
                 Identifier(name, span) => {
+                    if parameter_complete and not error {
+                        .error("paramters should be separated by a comma or a newline", .current().span())
```

Comment:
Let's make this a "expected <thing>" with that as a hint instead; same for the other commit

Reply:
![Screenshot 2022-07-30 at 00 05 14](https://user-images.githubusercontent.com/49019259/181823407-c79b2a1e-7c27-4ed1-b34d-be4f8b44859b.png)

Currently looks something like this, should I make any changes to this?